core/domain/html_cleaner.py

```diff
@@ -146,8 +146,51 @@ def get_rte_components(html_string):
     return components
 
 
+# Replace list to escape and unescape html strings.
+REPLACE_LIST = [
+    ('&', '&amp;'),
+    ('"', '&quot;'),
+    ('\'', '&#39;'),
+    ('<', '&lt;'),
+    ('>', '&gt;')
+]
+
+
+def escape_html(html_data):
+    """This functions converts html to escaped format.
+
+    Args:
+        html_data: str. HTML string to be escaped.
+
+    Returns:
+        str. Escaped HTML string.
+    """
+    for replace_tuple in REPLACE_LIST:
```

Comment:
Minor nit: suggest follow the same naming convention as `unescape_html` below, using the `escaped_html_data` and `unescaped_html_data` variable names.